lib/index.js

```diff
@@ -56,6 +56,23 @@ function start (config) {
     }
   }
 
+  function safeParse(line) {
+    try {
+      return JSON.parse(line);
+    }
+    catch (err){
+        return {
+          level: 30,
+          wasRaw: true,
+          time: `${(new Date()).getTime()/1000|0}`,
+          tags: ['info'],
+          data: {
+            msg: line
```

Comment:
We should make this configurable or at the very least it should follow what Pino [out of the box](https://github.com/pinojs/pino#usage) would return if we called `logger.info(line)`. 
```
{
   level: 30,
   time: ...
   msg: line
}
```

Reply:
We are using hapi-pino: https://github.com/pinojs/hapi-pino
And I was matching the relevant section here:
[mergeHapiLogData] - When enabled, Hapi-pino will merge the data received from Hapi's logging interface (server.log(tags, data) or request.log(tags, data)) into Pino's logged attributes at root level. If data is a string, it will be used as the value for the msg key. Default is false, in which case data will be logged under a data key. E.g.
`server.log(['info'], {hello: 'world'})`

```// with mergeHapiLogData: true
{ level: 30, hello: 'world', ...}

// with mergeHapiLogData: false (Default)
{ level: 30, data: { hello: 'world' }}```

Reply:
It does make sense that it should be configurable. Would it make sense to get the primary bug fixed using the default option, then swing back with a second PR to make it configurable.

Reply:
At the minimum, we need a patch release that people can pick up as easily as possible. Making it configurable in a different PR and releasing as a feature would be excellent followup.

Reply:
I'm mostly pointing out that if `mergeHApiLogData` is `false`, then
`server.log(['info'], 'Hello world')`
logs
`{ level: 30, data: 'Hello world', ...}`
you'd have to log `server.log(['info'], {msg: 'Hello world'})` to get the output you have above. 

However, I'm fine with just patching it as is and making a followup PR to make it configurable.

Reply:
Ah I see what you are saying. It should be 
`data: line`
not 
`data: {
msg: line
}`

Reply:
I fixed it as you suggested. @qswitcher